Cargo.toml

```diff
@@ -489,9 +489,9 @@ alloy-rpc-client = { version = "1.0.16", default-features = false }
 alloy-rpc-types = { version = "1.0.16", features = ["eth"], default-features = false }
 alloy-rpc-types-admin = { version = "1.0.16", default-features = false }
 alloy-rpc-types-anvil = { version = "1.0.16", default-features = false }
-alloy-rpc-types-beacon = { version = "1.0.16", default-features = false }
+alloy-rpc-types-beacon = { version = "1.0.17", default-features = false }
 alloy-rpc-types-debug = { version = "1.0.16", default-features = false }
-alloy-rpc-types-engine = { version = "1.0.16", default-features = false }
+alloy-rpc-types-engine = { version = "1.0.17", default-features = false }
```

Comment:
we always need to increase these in lockstep, so we should bump all alloy deps here from 1.0.16 to 1.0.17